coordinator/src/services/translation/prescription/index.ts

```diff
@@ -184,9 +180,7 @@ function convertPrescriptionPertinentInformation2(
   const pertinentInformation2 = []
 
   for (let i = 0; i < fhirMedicationRequests.length; i++) {
```

Comment:
This function is effectively a map over the incoming FHIR medication requests, so we can probably refactor things slightly to avoid the clunky `for` loop and the temporary array:
```typescript
return fhirMedicationRequests.map((medicationRequest, index) => {
    const patientInfoText = index === 0 ? extractText(fhirCommunicationRequests) : []
    const pertinentLineItem = convertMedicationRequestToLineItem(
      medicationRequest,
      repeatNumber,
      patientInfoText
    )
    return new prescriptions.PrescriptionPertinentInformation2(pertinentLineItem)
  })
```

Reply:
Ah, didn't know about the index parameter. Not my code, but I'll look at changing it